src/router/index.js

```diff
@@ -1,6 +1,7 @@
 import { createRouter, createWebHistory } from 'vue-router'
 import Home from '../views/Home.vue';
 import TomMenu from '../views/locations/tom/Menu.vue';
+import DanMenu from '../views/locations/dan/DanMenu.vue';
```

Comment:
I think this is where your issues are stemming from.  This import path is off a little bit because `/locations/dan` is actually `/locations/Dan` in the project directory--capitalization matters.  It's looking for the lowercase `dan` folder and can't find it.

You could capitalize the `Dan` folder in this line, but what I'd actually recommend is to rename the folder to be lowercase in the project directory to match the `tom` folder.  It doesn't matter too much one way or the other, but in general you always want to err towards being consistent with what already exists when naming/formatting things.

Reply:
BTW, I was able to see that this is the problem right away by looking at the terminal window that I ran `npm run serve` in after pulling in your branch.  If the code compilation isn't successful because of an error like this being thrown, that terminal window will generally show you the error, which will hopefully contain some kind of info message that will help you deduce the problem you're having.  In this case it was:
```
 ERROR  Failed to compile with 1 error                                                                                                                                12:01:53 PM
This relative module was not found:

* ../views/locations/dan/DanMenu.vue in ./src/router/index.js
```

This is a good first place to check if your site isn't loading properly and you're not sure why.  And if there's nothing in the terminal window then you can check the Console tab of the Chrome developer tools as well.